lib/report_email_sender.rb

```diff
@@ -0,0 +1,41 @@
+class ReportEmailSender
```

Comment:
Really? I've not seen that – link? (Want to read up a bit before we decide what to do).

Sticking with ActionMailer is nice as we get its ecosystem of support (e.g. SendGrip has ActionMailer hooks built in)